external/base-image/Dockerfile

```diff
@@ -62,13 +62,15 @@ RUN set -ex; \
   mkdir $FLINK_HOME/flink-web-upload; \
   mv $FLINK_HOME/opt/flink-queryable-state-runtime_$SCALA_VERSION-$FLINK_VERSION.jar $FLINK_HOME/lib; \
   chmod -R 777 $FLINK_HOME &&\
-  chmod -R 777 $SPARK_HOME
+  chmod -R 777 $SPARK_HOME && \
```

Comment:
Minor: Do we need "everything to everybody" file access here? 755 is often considered "enough' for most applications.

Reply:
will change ..

Reply:
done ..